### PR TITLE
New kustomize config for BMO standalone, needed with kind when ironic…

### PR DESCRIPTION
**Pull request by mikkosest**
… outside cluster. This commit contains config that can be used when kind is wanted to be used as option for minikube

Completes #472

### CODE REVIEW

**Comment by maelk** [comment]
You probably don't need to create this ironic_outside folder. There is already a no_ironic folder in the operator folder. I believe this would be enough

**Comment by mikkosest** [comment]
> You probably don't need to create this ironic_outside folder. There is already a no_ironic folder in the operator folder. I believe this would be enough
Fixed

**Comment by mikkosest** [comment]
> looks good.
> btw, what is the effect of having IRONIC_FAST_TRACK set to false (was true)?
IRONIC_FAST_TRACK is currently broken so it should be set false everywhere

File: deploy/ironic_ci.env [hunk 1]
```diff
@@ -6,4 +6,4 @@ DEPLOY_RAMDISK_URL=http://172.22.0.1:6180/images/ironic-python-agent.initramfs
 IRONIC_ENDPOINT=http://172.22.0.1:6385/v1/
 IRONIC_INSPECTOR_ENDPOINT=http://172.22.0.1:5050/v1/
 CACHEURL=http://172.22.0.1/images
-IRONIC_FAST_TRACK=true
+IRONIC_FAST_TRACK=false
```
Comment (maelk) [inline deploy/ironic_ci.env]: Could the script that starts ironic outside of the cluster rather use deploy/ironic-outside-config/ironic_bmo_configmap.env ? that would avoid duplication

Comment (mikkosest) [reply deploy/ironic_ci.env]: Fixed

**Pull request merged**